logger.go

```diff
@@ -0,0 +1,21 @@
+package producer
+
+import (
+	"fmt"
+)
+
+// Logger represents a simple interface used by kinesis-producer to handle logging
+type Logger interface {
+	Info(msg string, values ...Value)
+	Error(msg string, err error, values ...Value)
+}
+
+// Value represents a key:value pair used by the Logger interface
+type Value struct {
```

Comment:
I would rename this struct into something like `LogEntry`, since it's a bit confusing that it's under `producer` namespace.

Reply:
Sure. I moved it from the loggers package to top level but didn't rename,.